src/stores/PreferenceStore.ts

```diff
@@ -86,6 +87,7 @@ const DEFAULTS = {
         dragPanning: true,
         spectralMatchingType: SpectralType.VRAD,
         autoWCSMatching: WCSMatchingType.NONE,
+        transparentImageBackground: true,
```

Comment:
I think the default should be `false`, especially since the older schema doesn't have this field

Reply:
fixed